buildkite/src/Monorepo.dhall

```diff
@@ -23,17 +23,23 @@ let makeCommand : JobSpec.Type -> Cmd.Type = \(job : JobSpec.Type) ->
   let dirtyWhen = SelectFiles.compile job.dirtyWhen
   let trigger = triggerCommand "src/Jobs/${job.path}/${job.name}.dhall"
   in Cmd.quietly ''
-    if cat _computed_diff.txt | egrep -q '${dirtyWhen}'; then
-        echo "Triggering ${job.name} for reason:"
-        cat _computed_diff.txt | egrep '${dirtyWhen}'
-        ${Cmd.format trigger}
-    fi
+    case "$BUILDKITE_PIPELINE_TYPE" in
```

Comment:
what happens if `BUILDKIT_PIPELINE_TYPE` is not `""`, `pull_request`, or `stable`?

Reply:
Right now, no pipelines would be triggered. Would you like the default behavior to print an error message and exit with an error code?

Reply:
I think we should, yeah 👍

Reply:
ah right this is in a bash context